### PR TITLE
Slug: URL-safe slug generator

### PR DESCRIPTION
**Pull request by aantron**
In summary,

```ocaml
Slug.slugify "Some page title"
```

returns

```ocaml
"some-page-title"
```

which is more suitable for use in URLs. Slug offers several options for tweaking the exact behavior.

<br>

From the README:

> A URL slug is the part of a URL or link that comes after the domain extension.
>
> In websites the keyword used for your URL slug can be used to SEO optimize the URL by showing Google the structure of your site and the contents of the page in question.

This library turns title into URL-safe slug with support for non-latin characters.

This library uses algorithm and data from [node-slugify](https://github.com/simov/slugify)

### CODE REVIEW

**Comment by aantron** [comment]
cc @thangngoc89

File: packages/slug/slug.1.0.0/opam
Comment (mseri) [inline packages/slug/slug.1.0.0/opam:8]:
```suggestion
  [ "dune" "subst" ] {dev}
```

File: packages/slug/slug.1.0.0/opam
Comment (mseri) [inline packages/slug/slug.1.0.0/opam:11]:
```suggestion
depends: [
   "ocaml" {>= "4.05"}
```
Due to `Unbound value Hashtbl.find_opt`

File: packages/slug/slug.1.0.0/opam
Comment (mseri) [inline packages/slug/slug.1.0.0/opam:12]:
```suggestion
  "dune" {>= "2.0"}
```

**Review by mseri — CHANGES_REQUESTED** [review]
A few things to update. Can you make the changes upstream?

File: packages/slug/slug.1.0.0/opam
Comment (avsm) [inline packages/slug/slug.1.0.0/opam:24]:
```suggestion
description: """
A URL slug is the part of a URL or link that comes after the domain extension.

In websites the keyword used for your URL slug can be used to SEO optimize the URL by showing Google the structure of your site and the contents of the page in question.
"""
```

From the README -- this will help make the metadata on central docs sites much better

Comment (aantron) [reply packages/slug/slug.1.0.0/opam]: Thanks!

File: packages/slug/slug.1.0.0/opam
Comment (mseri) [inline packages/slug/slug.1.0.0/opam:21]:
```suggestion
  "re" {>= "1.7.2"}
```
This is where the `Re.X` modules where introduced

File: packages/slug/slug.1.0.0/opam
Comment (mseri) [inline packages/slug/slug.1.0.0/opam:18]:
```suggestion
  "uunf" {>= "1.0.0"}
```

The `Uunf.string` module was introduced here I think

**Review by mseri — CHANGES_REQUESTED** [review]

**Comment by thangngoc89** [comment]
I updated upstream with all the suggested changes

**Comment by mseri** [comment]
Can you please add a license field with the correct license in spdx format?
After that, this can be merged

**Comment by thangngoc89** [comment]
@mseri I added it upstream here but I don't have write access to this PR 
https://github.com/thangngoc89/ocaml-slug/commit/e4d8f79648c202977e46524410d5b55d6c0c6760

File: packages/slug/slug.1.0.0/opam
Comment (mseri) [inline packages/slug/slug.1.0.0/opam:7]:
```suggestion
license: "MIT"
build: [
```

**Comment by mseri** [comment]
Good to merge now!

**Comment by mseri** [comment]
Consider announcing the release on discuss.ocaml.org :)

**Comment by thangngoc89** [comment]
Thank you very much.

**Pull request merged**
